docs/core/diagnostics/dotnet-symbol.md

```diff
@@ -96,4 +96,4 @@ dotnet-symbol --host-only --debugging <dump file path>
 ## See also
 
 * [Debugging with symbols](/windows/win32/dxtecharts/debugging-with-symbols)
-* [Portable PDBs](https://github.com/dotnet/core/blob/master/Documentation/diagnostics/portable_pdb.md)
+* [Portable PDBs](https://github.com/dotnet/core/blob/main/Documentation/diagnostics/portable_pdb.md)
```

Comment:
The link is working but wondering if it should point to the following doc page:

https://docs.microsoft.com/en-us/dotnet/core/diagnostics/symbols